nilearn/plotting/displays.py

```diff
@@ -218,6 +236,89 @@ def draw_position(self, size, bg_color, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def _add_graph_nodes(self, nodes_coords, node_color, node_size, **kwargs):
+        """Plot graph nodes"""
+        if 's' in kwargs:
```

Comment:
Maybe these translation from node_color and node_size to arguments for scatter should be done in the plot_connectome function, and not here. The reason is that the notion of 'node' seems to me somewhat external to the function. It makes sense only when there are nodes and edges. I think that I would push for removing the term "node" from the code of this function.